frontend/test/metabase-smoketest/admin.cy.spec.js

```diff
@@ -0,0 +1,381 @@
+      .get(".Button")
+      .contains("Save")
+      .click();
+    cy.findByText("Not now").click();
+
+    // =================
+    // should create a new dashboard with the previous questions
+    // =================
+
+    // New dashboard
+    cy.get(".Icon-add").click();
+    cy.findByText("New dashboard").click();
+
+    cy.contains("Which collection should this go in?");
+
+    cy.findByLabelText("Name").type("Demo Dash 2");
+    cy.findByText("Create").click();
+
+    // Adding saved questions
+    cy.get(".Header-buttonSection").click("left");
+    cy.findByText("Order Totals by State").click();
+    cy.get(".Header-buttonSection").click("left");
+    cy.findByText("Orders Over Time").click();
+
+    cy.contains("Order Totals by State");
+    cy.contains("Orders Over Time");
+
+    cy.findByText("Save").click();
+
+    // ================
+    // should add a new user
+    // =================
+
+    // Sets up route
+    cy.server();
+    cy.route({
+      method: "POST",
+      url: "/api/user",
+    }).as("createUser");
+
+    // Navigates through admin pages
+    cy.get(".Nav")
+      .children()
+      .last()
+      .children()
+      .last()
+      .click();
+    cy.findByText("Admin").click();
+
+    cy.contains("Metabase Admin");
+    cy.contains("dashboard").should("not.exist");
+
+    cy.findByText("People").click();
+
+    cy.contains("Groups");
+
+    // Inputs user info (first modal)
+    cy.findByText("Add someone").click();
+    cy.findByLabelText("First name").type(new_user.first_name);
+    cy.findByLabelText("Last name").type(new_user.last_name);
+    cy.findByLabelText("Email").type(new_user.username);
+    cy.get(".ModalBody")
+      .find(".Icon-chevrondown")
+      .click();
+    cy.findByText("English").click();
+    cy.findByText("Create").click();
+
+    cy.wait("@createUser").then(xhr => {
+      cy.wrap(xhr.request.body.password).as("password");
+    });
+
+    //  Password confirmation(second modal)
+    cy.contains("has been added");
+    cy.findByText("Done").click();
+
+    cy.contains(new_user.username);
+
+    // ****************
+    // *** NEW USER ***
+    // *****************
+
+    // =================
+    // New user can sign in
+    // =================
+
+    signOut();
+    cy.get("@password").then(pass => {
+      cy.visit("/");
+      cy.findByLabelText("Email address").type(new_user.username);
+      cy.findByLabelText("Password").type(pass);
+      cy.findByText("Sign in").click();
+      cy.contains(new_user.first_name);
+
+      // =================
+      // should see questions currently in the "Our Analytics" collection as user
+      // =================
+
+      cy.findByText("Browse all items").click();
+      cy.contains("My personal collection");
+
+      // =================
+      // should see dashboard in the "Our Analytics" collection as user
+      // =================
+
+      cy.findByText("Dashboards").click();
+      cy.contains("Demo Dash 2");
```

Comment:
Let's validate that the user can click into both the Demo Dash and an admin-created question and they show the expected results.